.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,23 +1,65 @@
 ## Overview
-
+<!-- What changes did you make? Provide a summary of the updates included in this pull request -->
 
 ## Functional Changes
+<!-- Put an X next to an item -->
+[ ] New map or map update
+[ ] New Feature
+[ ] Feature update or enhancement
+[ ] Code Cleanup or refactor
+[ ] Code Removal
+[ ] Bug fix
```

Comment:
"GitHub actions" is still in beta AFAIK.
Once its released, we could create an action that auto-labels an issue or pull Request based on such checked lists ^^

Reply:
The labels are kinda nice, particularly for historical reference and when filling in release notes. I hope they help the 'observers' to know what a PR is about. Would be cool to see the automation for the label in place when it arrives